common/app/views/support/cleaner/SoundcloudHelper.scala

```diff
@@ -7,7 +7,7 @@ import org.jsoup.nodes.{Document, Element}
 
 // There are two element types that have been found to contain Soundcloud embeds.
 // These are: element-audio and element-embed
-object AmpSoundcloud {
+object SoundcloudHelper {
```

Comment:
Hey like that this is now an object! Can we move it into the dotcomponents folder as it is only used there/relevant to that now?

Reply:
Sure. You are right it's only used by `dotcomrendering`, didn't notice that.